src/App.js

```diff
@@ -157,7 +157,7 @@ class App extends Component {
           <button id="zoom-out" onClick={() => this.zoom(-0.2)}>-</button>
         </div>
         <div>
-          <PdfLoader url={url} beforeLoad={<Spinner />}>
+          <PdfLoader url={url} beforeLoad={<Spinner />} cMapUrl={"https://cdn.jsdelivr.net/npm/pdfjs-dist@2.6.347/cmaps/"} cMapPacked={true}>
```

Comment:
Same as this version. https://github.com/hungriesthippo/pdf-viewer/blob/8fb589d17c6080bd9a10e310ec806a47b5b1ff11/package-lock.json#L10489